frontend/src/components/summary/group/ECEWESummary.vue

```diff
@@ -49,145 +49,125 @@
-              >
-                <v-row no-gutters>
-                  <v-col cols="12">
-                    <span class="summary-label pt-3">
-                      I confirm our organization/facilities has reached an agreement with the union to amend the
-                      collective agreement(s) in order to implement the ECE Wage Enhancement.
-                    </span>
-                    <v-text-field
-                      placeholder="Required"
-                      :model-value="getYesNoValue(ecewe?.confirmation)"
-                      class="summary-value"
-                      density="compact"
-                      flat
-                      variant="solo"
-                      hide-details
-                      readonly
-                      :rules="rules.required"
-                    />
-                  </v-col>
-                </v-row>
-              </v-col>
-            </v-row>
+            <div v-if="organizationProviderType === ORGANIZATION_PROVIDER_TYPES.GROUP">
+              <v-row v-if="ecewe?.optInECEWE == 1" no-gutters>
+                <v-col cols="12">
+                  <span class="summary-label pt-3">
+                    Do any of the ECE employees at any facility in your organization belong to a union
+                  </span>
+                  <v-text-field
+                    placeholder="Required"
+                    :model-value="getYesNoValue(ecewe?.belongsToUnion)"
+                    class="summary-value"
+                    density="compact"
+                    flat
+                    variant="solo"
+                    hide-details
+                    readonly
+                    :rules="rules.required"
+                  />
+                </v-col>
+              </v-row>
+              <v-row v-if="!facilityInformationExists()" no-gutters>
+                <v-col v-if="languageYearLabel != programYearTypes.HISTORICAL && ecewe?.optInECEWE == 1" cols="12">
+                  <span class="summary-label pt-3">
+                    Are you a public sector employer, as defined in the Public Sector Employers Act?
+                  </span>
+                  <v-text-field
+                    placeholder="Required"
+                    :model-value="getYesNoValue(ecewe?.publicSector)"
+                    class="summary-value"
+                    density="compact"
+                    flat
+                    variant="solo"
+                    hide-details
+                    readonly
+                    :rules="rules.required"
+                  />
+                </v-col>
+                <v-col v-if="showApplicableSector" cols="12">
+                  <v-row no-gutters>
+                    <v-col cols="12">
+                      <span class="summary-label pt-3">Applicable Sector:</span>
+                      <v-textarea
+                        placeholder="Required"
+                        :model-value="getSectorValue(ecewe?.applicableSector)"
```

Comment:
I believe this can just be written as a computed method. I.e. sectorValue.

Reply:
agreed - done